cargo-verify/src/main.rs

```diff
@@ -565,12 +565,15 @@ fn build(opt: &Opt, package: &str, target: &str) -> CVResult<PathBuf> {
     let backend_lc = format!("{}", opt.backend).to_lowercase();
     let runtime = format!("{}/runtime/rvt-{}.bc", rvt_dir, backend_lc);
     let runtime = PathBuf::from(&runtime);
+    let simd_emulation = format!("{}/simd_emulation/simd_emulation.bc", rvt_dir);
+    let simd_emulation = PathBuf::from(&simd_emulation);
```

Comment:
Everything from `let rvt` above, down to this line (including) can be rewritten more idiomatically like this:
```rust
    let rvt_dir = PathBuf::from( std::env::var("RVT_DIR")? );
    let runtime = rvt_dir.clone()
        .append("runtime")
        .append(format!("rvt-{}.bc", opt.backend.to_string().to_lowercase()));
    let simd_emulation = rvt_dir.clone()
        .append("simd_emulation")
        .append("simd_emulation.bc");
```

Reply:
Done